app/pages/home/homePageController.ts

```diff
@@ -30,17 +30,22 @@ export class HomePageController {
         group: 'servicecatalog.k8s.io',
         resource: 'serviceclasses'
       }, {}, (resources: any) => {
-        this.ctrl.services = resources.by("metadata.name"); ;
-        this.ctrl.applications = this.constants.REDHAT_APPLICATIONS;
-        this.ctrl.categories = this.constants.SERVICE_CATALOG_CATEGORIES;
-        this.ctrl.loading = false;
+        this.ctrl.catalogServices = resources.by("metadata.name");
       });
     } catch (e) {
       console.log("Error Loading serviceclasses from Service Catalog.k8s.io: " + e);
-      this.ctrl.services = [];
-      this.ctrl.applications = this.constants.REDHAT_APPLICATIONS;
-      this.ctrl.categories = this.constants.SERVICE_CATALOG_CATEGORIES;
-      this.ctrl.loading = false;
+      this.ctrl.catalogServices = {};
     }
+
+    try {
```

Comment:
Hi, yes, getting a lot of runtime errors when/if 'addtionalServers' isn't setup in config.js, or other settings in the pipeline are not configured correctly (etc/hosts).